### DISCUSSION TITLE
ISC Services Directory

### DISCUSSION THREAD

**Discussion by claredillon** [InnerSourceCommons/InnerSourceMarketing#170] [General]
We are moving forward with the long-planned implementation of an InnerSource Commons Services Directory. 

You can find out more information about the goals and principles [here](https://docs.google.com/document/d/1WrRxs9Gz_OB996C6bzTjSvGKOm7EB9omEtN5YZJmLWo/edit?usp=sharing). 

We welcome:
- And feedback / comments or questions about the plans, principles and guidelines. 
- Anyone who is interested in participating or being featured in the first iteration of the Services Directory.

Look forward to hearing from you! :)

**Comment by zkoppert** [comment]
GitHub would like to participate! Here is our relevant information:

## Your logo for the website
<img width="200px" src="https://github.githubassets.com/images/modules/logos_page/GitHub-Logo.png">

or anything from https://github.com/logos

## A description of 100 words or less of your InnerSource tool or service.
GitHub offers a services engagement called  an "InnerSource Workshop" for organizations looking to start or improve their InnerSource practice. In this engagement, a GitHub professional will set baseline knowledge, facilitate discussion to establish alignment, and lead a practical workshop, resulting in the creation of a custom-tailored roadmap to implement innersource governance, administration, and culture.

## A link to a corresponding description of your InnerSource tool or service on your website.
https://services.github.com/catalog/#InnerSource%20Theory%20for%20Teams

## Contact details
https://services.github.com/#contact

**Comment by mishari** [comment]
Zymple would like to participate! Here is our relevant information:

## Your logo for the website
![ZYMPLE logo+name_RGB_withouttagline](https://user-images.githubusercontent.com/503018/165270473-08780480-ef0b-49df-805e-41907aaf8308.png)

## A description of 100 words or less of your InnerSource tool or service.
Zymple is an experienced tech consultancy based in South East Asia with expertise in open source processes, remote first workflows, and InnerSource. Our clients range from small teams of less than 10 employees to unicorns.

## A link to a corresponding description of your InnerSource tool or service on your website.
https://zymple.biz

## Contact details
hello@zymple.biz

**Comment by emiliogaleanog** [comment]
Bitergia would like to participate! Here is our relevant information:
### Your logo for the website
![bitergia_logo_text](https://user-images.githubusercontent.com/100841981/169468756-50854e0a-125b-4577-bf25-e92a39b52455.png)

### A description of 100 words or less of your InnerSource tool or service.
Bitergia offers InnerSource-as-a-Service to help at any stage of your InnerSource journey.  Advancing from where you are today, Bitergia offers a mix of different tools such as a maturity model, an AS-IS assessment, a capabilities model, and InnerSource metrics dashboards. Bitergia builds your turnkey InnerSource Program Office (ISPO), bringing the people, skills, and tools. Bitergia’s unique strength is in analytics, which is evident in the evidence-based consultancy work. We make it easier for you to bring the best open practices to suit your organization, strengthen communication and knowledge sharing across distributed teams and have full transparency, insights, and understanding using best-in-class metrics.

### A link to a corresponding description of your InnerSource tool or service on your website.
https://bitergia.com/innersource/

### Contact details
https://bitergia.com/contact.html

**Comment by conrogers** [comment]
GitLab would like to participate! Here is our relevant information:

### Your logo for the website
![gitlab-logo-100](https://user-images.githubusercontent.com/14933250/169554691-ae0c5e68-cac4-4bc6-baa5-95169af17411.png)

Or any from here: https://about.gitlab.com/press/press-kit/

### A description of 100 words or less of your InnerSource tool or service.
InnerSource is core to the GitLab values of "Everyone can Contribute". As such, we are very experienced in the best ways to source ideas and contributions across enterprises. We offer both free and paid trainings for organizations looking to securely InnerSource and bring out innovation from all corners of the company. Our free workshop is led by a Technical Account Manager for accounts that qualify. You can also do a professional services led training.

### A link to a corresponding description of your InnerSource tool or service on your website.
For the professional services led training: https://about.gitlab.com/solutions/innersource/
For more information on the TAM led workshops: https://about.gitlab.com/handbook/customer-success/workshops/innersource-workshop.html

### Contact details
https://about.gitlab.com/sales/
Or if you're a GitLab enterprise customer, reach out to your account team.
